### PR TITLE
Update UG and DG for Submission

### PR DESCRIPTION
**Pull request by sharpstorm**
## Changes
This PR contains the final version of the user guide and developer guide for submission. See live [here](https://sharpstorm.github.io/tp/DeveloperGuide.html).

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -722,7 +788,7 @@ Action | Format, Examples
 **Find Appointment** | `findappt KEYWORD [MORE_KEYWORDS] [by/(person OR name)]` <br> e.g `findappt Meeting`
 **Help** | `help`
 **Export CSV** | `exportcsv`
-**Import CSV** | `importcsv f/FILEPATH [n/COLUMNNUM] [p/COLUMN_PERSON] [e/COLUMN_EMAIL] [a/COLUMN_ADDRESS] [t/COLUMN_TAGS]` <br> e.g., `importCSV n/2 p/3 e/5 a/6 t/4`
+**Import CSV** | `importcsv f/FILEPATH [n/COLUMNNUM] [p/COLUMN_PERSON] [e/COLUMN_EMAIL] [a/COLUMN_ADDRESS] [t/COLUMN_TAGS]` <br> e.g., `importcsv n/2 p/3 e/5 a/6 t/4`
```
Comment (dandaandaaaaaan) [inline docs/UserGuide.md:791]: my bad on this but can help me change this to
```suggestion
**Import CSV** | `importcsv f/FILEPATH [n/COLUMNNUM] [p/COLUMN_PERSON] [e/COLUMN_EMAIL] [a/COLUMN_ADDRESS] [t/COLUMN_TAGS]` <br> e.g., `importcsv f/data/csvFile.csv n/2 p/3 e/5 a/6 t/4`
```

**Review by dandaandaaaaaan — CHANGES_REQUESTED** [review]
one minor thing for my part that i didnt notice but other than that lgtm thanks for making the effort to make this

**Review by KwanHW — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (sharpstorm) [reply docs/UserGuide.md]: Adopted in 40d26e910ce1b3157f5e11b3fc9b2bcc0c5092de.

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm

**Pull request merged**